src/DIRAC/FrameworkSystem/private/standardLogging/Logging.py

```diff
@@ -183,7 +172,7 @@ def registerBackend(self, desiredBackend, backendOptions=None, backendFilters=No
         """
         # Remove white space and capitalize the first letter
         desiredBackend = desiredBackend.strip()
-        desiredBackend = desiredBackend[0].upper() + desiredBackend[1:]
+        desiredBackend = desiredBackend.capitalize()
```

Comment:
Technically, these 2 are not exactly the same:

```
In [2]: s = "pIpPo"

In [3]: s.capitalize()
Out[3]: 'Pippo'

In [4]: s[0].upper() + s[1:]
Out[4]: 'PIpPo'
```

Reply:
Good point!